diagnostic.md

```diff
@@ -7,36 +7,45 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone git@github.com:kiraamaa/git-basics-diagnostic.git
+
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+Change directory to move into the the cloned directory. Then create a new branch (usually named response) and check it out.
```

Comment:
Correct. But to be clear, the branch should have a name that is appropriate to what you are doing. "response" makes sense for our diagnostics. But if, let's say, you're fixing a bug, the branch might be called "bugFix".